src/codesy.coffee

```diff
@@ -25,12 +25,12 @@ codesy.bid.get = (ajax_params) ->
   ajax_options.url = codesy.options.url +  '/bid/'
   $.ajax ajax_options
 
-codesy.bid.update = (form) ->
-  form = form or {}
+codesy.bid.update = ($form) ->
```

Comment:
Form needs to be a jquery object to use .serialize() and . att().  This was lost when moving the codesy.update function from the submit function.  Using the $ convention to emphasize this is a jquery object.